### ISSUE TITLE
A11y_AzureDataStudioSQL_Projects_Publish Projects_ScreenReader: NVDA does not announces name of database Combox 

### ISSUE THREAD

**Issue by Yash14j** [microsoft/azuredatastudio#14363]
### Actual:
NVDA does not announces name of database Combox. Nvda only announces Combobox collapse when navigated on it with the help of tab key.

Same issue is observed with Jaws

### Expected:
NVDA should announce name of Combox when navigated on it with the help of tab key.. Nvda should announce it as Database editbox collapsed.

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AzureDataStudioSQL_Feb2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG1.3.1;#NVDA;#Element:Combobox:#JAWS;#STP;#AINotInScope;

### Environment Details:
Application Name: Azure Data Studio
Version: 1.27.0-insider
Windows Version: Windows10
Screen reader: NVDA(2020.2), Jaws(2020.2008.24)

### Repro Steps:

1. Launch the application-Azure Database Project and install extension SQL Database Projects.
2. Navigate to SQL Database Project Home Screen with heading "Projects".
3. Tab to connections select your server and connect to it.
4. Tab to Projects tab an right click on any created project.
5. Tab till "Publish Project" and hit enter
6. Open nvda and tab to database combobox and check the result.

### User Impact:
Screen reader user will not get all the information Database combobox.

“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Comment by kisantia** [comment]
@Yash14j the name "Database" gets announced right after combo box. Is there something I'm misunderstanding?
![image](https://user-images.githubusercontent.com/31145923/111234781-8e1bfc80-85ac-11eb-9adf-485d97cf6572.png)

**Comment by kisantia** [comment]
closing because the combobox name is getting announced, as mentioned above. Please reopen if needed.

**Comment by Yash14j** [comment]
closing because the combobox name is getting announced, as mentioned above
#closed
